### PR TITLE
Update OS image in Anup's GPU VM

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**

### CODE REVIEW

**Comment by bgruening** [comment]
Looks good for me, but "no changes" looks unexpected.

File: instance_dedicated_gpu-alpha.tf [hunk 1]
```diff
@@ -3,7 +3,7 @@ variable "gpu-node-alpha-dns" {
 }
 
 data "openstack_images_image_v2" "gpu-node-alpha-image" {
-  name = "vggp-gpu-v60-j310-1fad751e0150-main"
+  name = "vggp-v60-gpu-j322-692e75a7c101-main"
```
Comment (bgruening) [inline instance_dedicated_gpu-alpha.tf:6]: why has the name shifted from gpu-v60 to v60-gpu?

Comment (sanjaysrikakulam) [reply instance_dedicated_gpu-alpha.tf:6]: I wondered the same. I copied the image from our [VGCN infrastructure](https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/main/resources.yaml#L7C12-L7C47). I will later look when was this image created and from next versions let's stick to one naming scheme.

**Comment by sanjaysrikakulam** [comment]
> Looks good for me, but "no changes" looks unexpected.

Yeah, unexpected. However, I think we can merge this.

**Pull request merged**